### PR TITLE
Initial port

### LINKED ISSUE #216: Port code to use girder 4 multinetjs
Depends on multinet-app/multinetjs#29.

### PR DESCRIPTION
**Pull request by naglepuff**
Closes #216

This PR makes the necessary changes to multimatrix for compatibility with the Girder4/Django API.

Since incoming data is different than the existing Flask-based backend, some light refactoring had to be done to accommodate these changes.

Also, authentication has been updated to use girder-oauth-client. Some configuration files were also updated.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -18,11 +18,11 @@ export default {
...
 
   setup() {
-    const { workspace, graph: networkName } = getUrlVars();
+    const urlVars = getUrlVars();
 
     store.dispatch.fetchNetwork({
-      workspaceName: workspace,
-      networkName,
+      workspaceName: urlVars.workspace,
+      networkName: urlVars.network,
```
[the remark this answers is not in the document]
Comment (JackWilb) [reply src/App.vue:25]: https://github.com/multinet-app/multinet-client/issues/90

File: src/store/index.ts [hunk 2]
```diff
@@ -319,63 +320,50 @@ const {
...
-        nodePromises.push(api.table(workspaceName, table, { offset: 0, limit: 1000 }));
-      });
-
-      // Resolve all node table promises and extract the rows
-      const resolvedNodePromises = await Promise.all(nodePromises);
-      let nodes: TableRow[] = [];
-      resolvedNodePromises.forEach((resolvedPromise) => {
-        nodes.push(...resolvedPromise.rows);
-      });
+      const nodeRows = await api.nodes(workspaceName, networkName, { offset: 0, limit: 300 });
```
Comment (JackWilb) [inline src/store/index.ts:353]: Does this function work across all the tables? That's pretty nice functionality

**Review by JackWilb — COMMENTED** [review]
Looks good. It's what I expected and I noticed some nice new functions that makes pulling data easier. Do we not need to bump the version of multinetjs for this to work?

File: src/store/index.ts (hunk 2 above)
Comment (naglepuff) [reply src/store/index.ts:353]: Yup. The new API can get all nodes for a network via `api.nodes`, and/or all edges via `api.edges`. You can also do this at the table level via `api.table`.

**Comment by naglepuff** [comment]
Bumped `multinet` to `0.21.3` as part of the initial port. See 0e58b3b.

**Review by JackWilb — APPROVED** [review]

**Pull request merged**